learner_pathway_progress/management/tests/test_update_all_pathways_progress.py

```diff
@@ -27,7 +27,7 @@
 
 
 @pytest.mark.django_db
-class TestUpdateLearnerPathwayProgress(TestCase):
+class TestUpdateAllPathwaysProgress(TestCase):
```

Comment:
We have 2 tables, `Pathway` and `LearnerPathway` is this test for both of these pathways?